yarn.lock

```diff
@@ -4966,9 +4970,296 @@ table@^3.7.8:
+
+"tachyons-skins@github:eggheadio/tachyons-skins":
+  version "4.0.2"
+  resolved "https://codeload.github.com/eggheadio/tachyons-skins/tar.gz/59502baf4292ecefed027ec7f92afab51cd8655a"
+
+tachyons-spacing@^6.0.3:
+  version "6.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-spacing/-/tachyons-spacing-6.0.4.tgz#54cbd6d902b2a36ab864110754ec9924e09935aa"
+
+tachyons-styles@^0.3.4:
+  version "0.3.4"
+  resolved "https://registry.yarnpkg.com/tachyons-styles/-/tachyons-styles-0.3.4.tgz#bd1c3fdf62b27df07e4be17a7d3adda2c89ef71d"
+
+tachyons-tables@^1.0.6:
+  version "1.0.7"
+  resolved "https://registry.yarnpkg.com/tachyons-tables/-/tachyons-tables-1.0.7.tgz#2f914bbdb07c7b232cb15392d716f4c965d0770a"
+
+tachyons-text-align@^3.0.3:
+  version "3.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-text-align/-/tachyons-text-align-3.0.4.tgz#444f09db34b11fd6e6787b665f12348f92f99ee9"
+
+tachyons-text-decoration@^4.0.3:
+  version "4.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-text-decoration/-/tachyons-text-decoration-4.0.4.tgz#47d724c83e9f0f6d68e301721fd4db6c321349bc"
+
+tachyons-text-transform@^4.0.3:
+  version "4.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-text-transform/-/tachyons-text-transform-4.0.4.tgz#7b009e4b1ff3b571053d0239f00b9c3bce67114d"
+
+tachyons-type-scale@^6.0.3:
+  version "6.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-type-scale/-/tachyons-type-scale-6.0.4.tgz#37f8bcd4fef64698f49d49acf791f4718604f0ba"
+
+tachyons-typography@^3.0.3:
+  version "3.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-typography/-/tachyons-typography-3.0.4.tgz#c428bb63ff65bdedb4a26d986ebd22e325e52b33"
+
+tachyons-utilities@^2.0.0:
+  version "2.0.0"
+  resolved "https://registry.yarnpkg.com/tachyons-utilities/-/tachyons-utilities-2.0.0.tgz#64df9486efc432924f20fd90b8d2c0fc1c295b75"
+
+tachyons-vertical-align@^4.0.3:
+  version "4.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-vertical-align/-/tachyons-vertical-align-4.0.4.tgz#afe7ddbf2963cfd2094a1b4d97ad974c555cd6d7"
+
+tachyons-visibility@^2.0.3:
+  version "2.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-visibility/-/tachyons-visibility-2.0.4.tgz#9a6d2ac1258d28aec081935dcef626857d3cf53f"
+
+tachyons-white-space@^4.0.4:
+  version "4.0.5"
+  resolved "https://registry.yarnpkg.com/tachyons-white-space/-/tachyons-white-space-4.0.5.tgz#cd51794d57ba19d3ed48d6ec2361f59edf095ecc"
+
+tachyons-widths@^5.1.0:
+  version "5.1.1"
+  resolved "https://registry.yarnpkg.com/tachyons-widths/-/tachyons-widths-5.1.1.tgz#abf61705b22630b9b10ce7874cddb26a5c01b023"
+
+tachyons-word-break@^3.0.3:
+  version "3.0.4"
+  resolved "https://registry.yarnpkg.com/tachyons-word-break/-/tachyons-word-break-3.0.4.tgz#242fef58c1650b998834ca5f67f21fff56ffa118"
+
+tachyons-z-index@^1.0.7:
+  version "1.0.8"
+  resolved "https://registry.yarnpkg.com/tachyons-z-index/-/tachyons-z-index-1.0.8.tgz#a853254917c9902ca2393da44ed2c093f148cbad"
```

Comment:
Would be cool if the dev dependencies in `tachyons-egghead` and `egghead-ui`  could be placed in `devDependencies` instead of `dependencies`, because it is currently adding a ton of uneeded weight to `node_modules` in consuming projects which aren't actually used after compilation / publishing; but I know there is that Heroku issue where it breaks with dev deps or whatever 😢